src/Swarrot/Broker/MessageProvider/MessageProviderInterface.php

```diff
@@ -13,6 +13,14 @@
      */
     public function get();
 
+    /**
+     * consume
```

Comment:
this phpdoc is totally useless. It should either describe what the method does or be removed (and yes, I know that `get` is already using such useless phpdoc)